CHANGES.rst

```diff
@@ -417,6 +420,9 @@ Bug Fixes
 
 - ``astropy.modeling``
 
+  - Fixed an issue with ``AiryDisk2D``, ``Sersic1D``, and ``Sersic2D``
+    models when used in compound models. [#4720]
```

Comment:
I'd be a bit more specific about the change. For example, something like "refactored the models to be able to combine them as classes as well as instances".